src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -1,7 +1,13 @@
 @import 'src/styles/common';
 @import 'constants';
 
-.defaultTranscriptsList {
+.header {
+  border-bottom: 1px solid $orange;
+  line-height: 2;
```

Comment:
What’s the purpose of the line-height here?

Reply:
Changed this to padding